docs/10.0/guides/building-and-testing/modules.md

```diff
@@ -9,718 +9,1079 @@ tags:
+| Core functionalities                                                       | `handsontable/base`<br>`handsontable/dist/handsontable.full.css`                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               | Required            |
+| [Cell editors](@/guides/cell-functions/cell-editor.md)                     | `AutocompleteEditor`<br>`BaseEditor`<br>`CheckboxEditor`<br>`DateEditor`<br>`DropdownEditor`<br>`HandsontableEditor`<br>`NumericEditor`<br>`PasswordEditor`<br>`SelectEditor`<br>`TextEditor`                                                                                                                                                                                                                                                                                                                                                                                                                                                  | Optional            |
+| [Cell renderers](@/guides/cell-functions/cell-renderer.md)                 | `baseRenderer`<br>`autocompleteRenderer`<br>`checkboxRenderer`<br>`htmlRenderer`<br>`numericRenderer`<br>`passwordRenderer`<br>`textRenderer`                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  | Optional            |
+| [Cell validators](@/guides/cell-functions/cell-validator.md)               | `autocompleteValidator`<br>`dateValidator`<br>`numericValidator`<br>`timeValidator`                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            | Optional            |
```

Comment:
On the linked source http://localhost:8080/docs/cell-validator/ there are 5 validators while in the table, we mention 4 of them. The same goes for the renderer. I think that this happens because those not included inherit from other editors/validators. But then according to http://localhost:8080/docs/cell-type/#usage `select` is only a defined `cellType` and it is missing from the table.

Reply:
@AMBudnik - right, the tables include only modules mentioned [here](https://handsontable.com/docs/modules/#modules-cheatsheet).

All in all, the following elements seem to be hidden somewhere in the imports (and I can't find them in the source code):
- `Handsontable.renderers.DropdownRenderer` (mentioned [here](https://handsontable.com/docs/cell-renderer/#cell-renderer))
- `Handsontable.renderers.DateRenderer` (mentioned [here](https://handsontable.com/docs/cell-renderer/#cell-renderer))
- `Handsontable.renderers.TimeRenderer` (mentioned [here](https://handsontable.com/docs/cell-renderer/#cell-renderer))
- `Handsontable.validators.DropdownValidator` (mentioned [here](https://handsontable.com/docs/cell-validator/#cell-validator))
- `Handsontable.cellTypes.select`(mentioned [here](https://handsontable.com/docs/cell-type/#available-cell-types))

@budnix - perhaps you can help us out: how do you import the elements above? I suppose they're included in some of the modules, but I can't locate them. Thanks!

Reply:
@kirszenbaum The mentioned "modules" are not really modules in terms of the modules released a year ago (8.3.0). They can not be imported because physically they do not exist. 

For example, the `dropdown` cell type (and others) underneath uses the renderer and validator from the other existing modules. Probably for the legacy support all registered cell types, renderers, validators, and editors including that aliased one are exported to the `Handsontable` global variable. And only there they are available.

For example for the Dropdown menu, the exported renderer [is the same](https://github.com/handsontable/handsontable/blob/master/handsontable/src/cellTypes/dropdownType/dropdownType.js#L10) as for Autocomplete
```js
Handsontable.renderers.DropdownRenderer === Handsontable.renderers.AutocompleteRenderer // true
```

About the `Handsontable.cellTypes.select` type I think this is a mistake. This cell type never esist.

Generally, the list of all modules can be found by listing the directories within each exported namespaces:
 * https://github.com/handsontable/handsontable/tree/master/handsontable/src/cellTypes;
 * https://github.com/handsontable/handsontable/tree/master/handsontable/src/editors;
 * https://github.com/handsontable/handsontable/tree/master/handsontable/src/plugins;
 * https://github.com/handsontable/handsontable/tree/master/handsontable/src/renderers;
 * https://github.com/handsontable/handsontable/tree/master/handsontable/src/validators;

Reply:
Thanks @budnix ! @AMBudnik - it seems that the new Modules page correctly lists all modules that can be imported, but there are discrepancies on the other side, on the following pages:

https://handsontable.com/docs/cell-renderer/#cell-renderer
https://handsontable.com/docs/cell-validator/#cell-validator
https://handsontable.com/docs/cell-type/#available-cell-types
https://handsontable.com/docs/select-cell-type/

I created a separate issue to deal with those: #9087

Reply:
Thank you for investigating this case